src/data/hymnalsConfig.tsx

```diff
@@ -0,0 +1,124 @@
+export type HymnalConfig = {
+    language: string;
+    title: string;
+    key: string;
+};
+
+export const getHymnalFileUrl = (language: string) =>
+    `https://raw.githubusercontent.com/TinasheMzondiwa/cis-hymnals/main/${language}.json`;
```

Comment:
We can also change to use supabase since files are hosted there but this works for now.

Reply:
Okay, cool